src/components/Layout/index.tsx

```diff
@@ -34,3 +46,25 @@ const Layout = (props: LayoutProps) => {
 // }
 
 export default Layout
+
+const temporaryAlerts: { id: string; content: ReactNode }[] = [
+  {
+    id: 'aave-issues-04-11-2023',
+    content: (
+      <span className='flex flex-col items-center text-center text-xxs'>
+        Aave has temporarily paused deposits and withdrawals on some assets until{' '}
+        <ExternalLink
+          href='https://app.aave.com/governance/proposal/?proposalId=358'
+          className='text-pt-teal transition hover:opacity-90 underline'
+          underline
+        >
+          proposal 358
+        </ExternalLink>{' '}
+        passes, due to a security issue. While no funds are at risk, the Prize USDC.e and Prize DAI
```

Comment:
This is V5 language- maybe just say the Optimism prize pool is unavailable in the meantime